pkg/utils/securitycontext.go

```diff
@@ -0,0 +1,211 @@
+package utils
+
+import (
+	"encoding/json"
+	"errors"
+	v1 "k8s.io/api/core/v1"
+	"keptn-contrib/job-executor-service/pkg/config"
+	"log"
+	"os"
+)
+
+// jobSecurityContextFilePath describes the path of the security config file that is defined in the deployment.yaml
+const jobSecurityContextFilePath = "/config/job-defaultSecurityContext.json"
```

Comment:
it would be better if these filenames were configurable from the outside of the package (those could easily be constants in main for example or maybe coming from a config file or env var)